src/System.Private.CoreLib/src/System/Threading/WaitHandle.cs

```diff
@@ -293,14 +299,14 @@ public static int WaitAny(WaitHandle[] waitHandles, int millisecondsTimeout)
 
```

Comment:
Talked offline: preference to keep all argument verification together in a single method.